assets/oauth-apiserver/oauth-apiserver-deployment.yaml

```diff
@@ -13,6 +13,7 @@ spec:
     matchLabels:
       app: openshift-oauth-apiserver
   progressDeadlineSeconds: 600
+  minReadySeconds: 30
```

Comment:
Since this deployment utilizes a `readinessProbe`, we should lower `minReadySeconds` to `15`.

Reply:
updated